src/game/server/gamecontext.cpp

```diff
@@ -2179,25 +2179,25 @@ void CGameContext::OnSayNetMessage(const CNetMsg_Cl_Say *pMsg, int ClientId, con
 		if(str_startswith_nocase(pMsg->m_pMessage + 1, "w "))
 		{
 			char aWhisperMsg[256];
-			str_copy(aWhisperMsg, pMsg->m_pMessage + 3);
+			str_copy(aWhisperMsg, pMsg->m_pMessage + sizeof("w "));
```

Comment:
```suggestion
			str_copy(aWhisperMsg, pMsg->m_pMessage + 1 + sizeof("w ") - 1);
```

It was unclear to me why the code was correct. I think this would improve the code. Adding a comment might be another option.

Reply:
If the code is not clear the commit description has it all. Your editors git blame feature should be able to display that if needed.

If you really need it in the code I would go for a ``len()`` macro to hide the -1 and then do a +1 but doing both +1 and -1 at once is a bit intense.

Reply:
What's wrong with doing both + 1 and -1? It shows the reader that something non-obvious is going on.

> Your editors git blame feature should be able to display that if needed.

Only until the line is reformatted. Blame view is not available by default everywhere, e.g. on GitHub, when merely reading code or trying to understand pull requests.

Reply:
Well if you think it's better go pr it. I don't think it's nicer but I won't stop you not that I can anyways.